index.js

```diff
@@ -1,20 +1,630 @@
+  }
+
+  //loop to keep player in the main corridor and adjoining rooms until they go down the elevator
+  async function mainCorridorLoop() {
+    inMainCorridor = true;
+    sceneThree.roomLooper();
+    userCommand = await ask("\nChoose an option >_".grey.bold);
+
+    while (inMainCorridor === true) {
+      if (userCommand === "1") {
+        elevatorLoop();
+      } else if (userCommand === "2") {
+        messHallLoop();
+      } else if (userCommand === "3" && inventory.includes("glow stick")) {
+        supplyRoomLoop();
+      } else if (userCommand === "3") {
+        console.log(
+          `
+You open the door the the Supply Room but it is pitch black. You reach over and 
+flick the light switch on. Nothing happens... There is no way you can look for  
+supplies her without a light source. You head back to the Main Corridor. 
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`.brightYellow
+        );
+        mainCorridorLoop();
+      } else if (userCommand.toLowerCase() === "i") {
+        console.log(
+          `
+[ Your current inventory is: ${inventory} ]
+`.brightRed
+        );
+      } else if (userCommand.toLowerCase() === "exit") {
+        quitGame();
+      } else {
+        console.log(
+          "\nThat is not a valid command. Please choose one of the numbered options"
+            .red
+        );
+      }
+      userCommand = await ask("\nChoose an option >_".grey.bold);
+    }
+  }
+
+  //function determining whether player has credentials to enter elevator or not
+  async function elevatorLoop() {
+    sceneThreeLocked = true;
+    while (sceneThreeLocked) {
+      if (inventory.includes("crowbar") && userCommand === "1") {
+        console.log(
+          `
+You use what little strength you have to pry open the stuck door with the 
+crowbar. You enter the elevator and push the button to send you down the 
+shaft towards the Engine Room. The door remains ajar as you hug the back 
+wall and decend the levels of your ship. The damaged cube grinds to a halt
+slightly missing it's mark but leaving enough room for you to crawl out into
+the Engine Room. Guess you're stuck down here now...
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`.brightYellow
+        );
+        sceneThreeLocked = false;
+        engineRoomLoop();
+      } else if (!inventory.includes("crowbar" && userCommand === "1")) {
+        console.log(
+          `
+You make your way to the elevator but the door is jammed. There is a two inch
+gap, but your attempts to pry it open with your fingers are in vain. You need 
+some leverage... You head back down the Main Corridor towards the other rooms.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`.brightYellow
+        );
+        sceneThreeLocked = false;
+        mainCorridorLoop();
+      }
+    }
+  }
+
+  //function giving user the opportunity to eat or to pick up an item in the mess hall
```

Comment:
Great functional programming patterns throughout the program. Love seeing everything broken out into separate functions where appropriate.